asset.go

```diff
@@ -62,6 +62,8 @@ const (
 	TRX    = NativeAsset("TRX")    // TRON
 	SEI    = NativeAsset("SEI")    // Sei
 	XRP    = NativeAsset("XRP")    // XRP
+	BASE   = NativeAsset("BASE")   // BASE
+	NOBL   = NativeAsset("NOBL")   // Noble Chain
```

Comment:
Let's use `NOBLE` instead.  NOBL is actually a different chain: https://coinmarketcap.com/currencies/nobleblocks/

Reply:
Make sense.